expo/tests/scheduler/test_manage_volunteer_opportunities.py

```diff
@@ -0,0 +1,274 @@
+from django.core.urlresolvers import reverse
+import nose.tools as nt
+from django_nose.tools import assert_redirects
+from unittest import TestCase
+from django.test import Client
+from tests.factories.gbe_factories import (
+    ProfileFactory,
+    RoomFactory
+)
+from scheduler.models import EventContainer
+from tests.functions.gbe_functions import (
+    grant_privilege,
+    login_as,
+)
+from tests.contexts import (
+    StaffAreaContext,
+)
+
+
+class TestEventList(TestCase):
+    view_name = 'manage_opps'
+
+    def setUp(self):
+        self.client = Client()
+        self.user = ProfileFactory.create().user_object
+        self.privileged_profile = ProfileFactory()
+        self.privileged_user = self.privileged_profile.user_object
```

Comment:
Again, I failed in my review, but this object is not required since grant_privilege takes either a user or a profile.